discord-roles.md

```diff
@@ -0,0 +1,72 @@
+| Probation              | New team members                  | Given to new team members.                                        |        :x:         |        None         |                None                |
+| Community Volunteers   | Community volunteer Helpers       | Given to community experts to help others.                        |        :x:         |        None         |                None                |
+| Former Member          | Old team members                  | Given to old team members to honor their contributions.           |        :x:         |        None         |                None                |
+| Snapshot Alarm         | Added via Discord Join            | Available for anyone to add in Channels & Roles.                  |        :x:         |        None         |                None                |
+| Announcement Pings     | Added via Discord Join            | Available for anyone to add in Channels & Roles.                  |        :x:         |        None         |                None                |
+| Dev Announcement Pings | Added via Discord Join            | Available for anyone to add in Channels & Roles.                  |        :x:         |        None         |                None                |
+| Snapshot Guesser       | Whoever guessed the snapshot time | Assigned by a team member.                                        |        :x:         |        None         |                None                |
+| All Vanity Roles       | All team members (optionally)     | The only roles with colors.                                       |        :x:         | 1 for each MC Color | 1 for each hoisted role + combined |
+| Nitro Booster          | Nitro boosters                    | Handled by the Discord integration.                               |        :x:         |        None         |         Nitro Booster Icon         |
+| All Bot Roles          | All bots                          | Discord-demanded integrations.                                    |        :x:         |        None         |                Gear                |
+
+### PIM
+PIM stands for Privileged Identity Management, its main goal is to provide a Just-In-Time way of giving users permissions when they need it.
+In a controlled and operational fashion. With or without approval of a second person.
+This ensures that roles with extreme rights, which when misused (either through hacking or for personal gain) would harm the server and the community, are not directly accessible via Discord alone.
+If an approval is needed for the role, it can only be granted by somebody other than the requester, even if the requester would normally be eligible to approve the role request.
+All team members will be eligible for any and all roles exposed via the bot.
```

Comment:
For proper markdown formatting, add a new line between each paragraph.
```suggestion
PIM stands for Privileged Identity Management. Its main goal is to provide a just-in-time way of giving users permissions when they need it, in a controlled and operational fashion, with or without approval of a second person.

This ensures that roles with extreme rights, which when misused (either through hacking or for personal gain) would harm the server and the community, are not directly accessible via Discord alone.

If an approval is needed for the role, it can only be granted by somebody other than the requester, even if the requester would normally be eligible to approve the role request.

All team members will be eligible for any and all roles exposed via the bot.
```